sign/sign_integration_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sign_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/release-sdk/sign"
+	"sigs.k8s.io/release-sdk/test"
+)
+
+func generateCosignKey(t *testing.T, getPassFunc cosign.PassFunc) string {
+	tempDir, err := os.MkdirTemp("", "k8s-cosign-keys-")
+	require.Nil(t, err)
+
+	keys, err := cosign.GenerateKeyPair(getPassFunc)
```

Comment:
I think we can pass `nil` here if https://github.com/sigstore/cosign/pull/1426 got merged / accepted.

Reply:
The PR got merged. I added a TODO about this, so we change it once a new cosign release is available.